bool is_active
geometry_msgs/Twist cmd_vel
